mpas_analysis/polar_regions.cfg

```diff
@@ -611,7 +611,125 @@ yLim = [-600., -5.]
 
 # comparison grid(s) on which to plot analysis
 comparisonGrids = ['latlon', 'arctic_extended', 'antarctic_extended',
-                   'subpolar_north_atlantic']
+                   'subpolar_north_atlantic', 'fris']
+
+[climatologyMapBSFArcticExtended]
+## options related to plotting Arctic climatologies of the barotropic
+## streamfunction (BSF)
+
+# colormap for model/observations
+colormapNameResult = blue-orange-div
+# whether the colormap is indexed or continuous
+colormapTypeResult = continuous
+# color indices into colormapName for filled contours
+# the type of norm used in the colormap
+normTypeResult = symLog
+# A dictionary with keywords for the norm
+normArgsResult = {'linthresh': 10., 'linscale': 0.5, 'vmin': -50., 'vmax': 50.}
+colorbarTicksResult = [-50., -40., -20., -10., 5., 0., 5., 10., 20., 40., 50.]
+# Adding contour lines to the figure
+contourLevelsResult = np.arange(-50., 50.1, 5.)
```

Comment:
@milenaveneziani, let's try this out and see how it works.  We can tweak things from there.

Reply:
I'm rerunning my SORRM test to see what this looks like.

Reply:
oh ok, thanks! But don't sweat it if it doesn't look good. We can tweak things later.

Reply:
The symmetric log colorbar doesn't add much.  I'm going to go back to linear.